azure/flow-runner-function/src/main/java/com/marklogic/dhf/azure/FunctionDhfFlow.java

```diff
@@ -38,20 +38,25 @@ public HttpResponseMessage run(
         } else {
             // Write the name to the message queue.
             msg.setValue(name);
-
-            runFlow();
-            return request.createResponseBuilder(HttpStatus.OK).body("Hello, " + name).build();
+            String runFlowMessage = "Flow <" + name + "> successfully executed."; //default message
```

Comment:
As part of this Jira, we are not actually running a valid flow. I plan to providing a valid/correct response as part of the next Jira 4532. As part of the current Jira 4531, we are always returning HTTP 200 with a status message (in this case it will always be exception message).